tests/natural_language/WriteGoodLintBearTest.py

```diff
@@ -127,22 +127,23 @@
                                                 settings={'allow_so_beginning':
                                                           False})
 
-WriteGoodLintBearWithIllusionTest = verify_local_bear(WriteGoodLintBear,
-                                                      valid_files=(
-                                                          good_file,
-                                                          weasel_file,
-                                                          wordy_file,
-                                                          thereIs_file,
-                                                          cliche_file,
-                                                          adverb_file,
-                                                          passive_file,
-                                                          bad_file,
-                                                          so_file,),
-                                                      invalid_files=(
-                                                          illusion_file,),
-                                                      settings={
-                                                        'allow_repeated_words':
-                                                            False})
```

Comment:
I believe this is the only line with an error for `WriteGoodLintBearWithIllusionTest`.

The easiest fix is to de-dent `False})` by four spaces.

Another way to solve it, without re-indenting everything, by creating a dict called `setting1` and using it here.

```
setting1 = {
    'allow_repeated_words': False,
}
...

                                                      settings=setting1)
```